### PR TITLE
Cincinnati: rename secrets

### PR DESCRIPTION
**Pull request by vrutkovs**
Secret names are excessively long

### CODE REVIEW

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Comment by vrutkovs** [comment]
/hold

Waiting for previous PR to land to re-run rehearsals

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.4.yaml [hunk 1]
```diff
@@ -52,37 +52,30 @@ tests:
 - as: cargo-test
   steps:
     pre:
-    - as: ensure-vendoring
-      commands: dist/vendor.sh && git diff --quiet
-      from: src
```
Comment (wking) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.4.yaml:57]: Why are we dropping this?  Not that I care all that much about the 4.4 branch anymore, just curious.

Comment (vrutkovs) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.4.yaml]: Right, lets leave it there

**Comment by wking** [comment]
[ci-operator-config][1] exploded:

```
non-unique image publication found: 
ERROR: output tag ocp-private/4.1-priv:src is promoted from more than one place: openshift-priv/cluster-api-actuator-pkg@release-4.1, openshift-priv/console@release-4.1
...
```

No idea what's up with that.

/retest

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/16807/pull-ci-openshift-release-master-ci-operator-config/1371507280893186048

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.8.yaml [hunk 2]
```diff
@@ -84,17 +84,17 @@ tests:
     - as: cargo-test
       commands: env CINCINNATI_CI_DOCKERJSON_PATH="${SHARED_DIR}/${CINCINNATI_CI_DOCKERJSON_FILE}" CINCINNATI_TEST_CREDENTIALS_PATH="${SHARED_DIR}/${CINCINNATI_TEST_CREDENTIALS_FILE}" dist/cargo_test.sh
       credentials:
-      - mount_path: /usr/local/ci-secrets/cincinnati-secrets-cincinnati-credentials
-        name: cincinnati-secrets-cincinnati-credentials
+      - mount_path: /usr/local/ci-secrets/cincy-credentials
+        name: cincy-credentials
```
Comment (wking) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.8.yaml:88]: [cargo-test-4.8][1]:

```
dist/cargo_test.sh: line 98: type: cargo: not found
dist/cargo_test.sh: line 98: type: docker: not found
error: could not find any of execute_native execute_docker in PATH
```

Dunno what's up with that.

/retest

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/16807/rehearse-16807-pull-ci-openshift-cincinnati-release-4.8-cargo-test/1371507519297425408#1:build-log.txt%3A32

**Comment by vrutkovs** [comment]
Not sure why release-4.x branch tests are failing (except 4.4), lets focus on master now and revisit this later?

/hold cancel

**Pull request merged**